lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb

```diff
@@ -59,8 +59,8 @@ class SmartStandalonePatientAppGroup < Inferno::TestGroup
         # Background
 
         The [Standalone
-        Launch](http://hl7.org/fhir/smart-app-launch/#standalone-launch-sequence)
-        Sequence allows an app, like Inferno, to be launched independent of an
+        Launch Sequence](http://hl7.org/fhir/smart-app-launch/app-launch.html#launch-app-standalone-launch)
```

Comment:
Likewise, these all need to point to SMART App Launch v1. The links in the SMART App Launch test kit also need to be updated.

Reply:
Okay, so would this be correct: http://hl7.org/fhir/smart-app-launch/1.0.0/index.html#standalone-launch-sequence

Reply:
Yes.